newIDE/app/src/Export/Builds/BuildCard.js

```diff
@@ -64,46 +114,239 @@ export const BuildCard = ({
+        ),
+        rawError: error,
+        errorId: 'build-delete-error',
+      });
+      setGameUpdating(false);
+    }
+  };
+
   return (
-    <Card>
-      <CardHeader
-        title={build.id}
-        subheader={
-          <Line alignItems="center" noMargin>
-            {getIcon(build.type)}
-            <Trans>
-              {formatBuildText(build.type)} -{' '}
-              {format(build.updatedAt, 'yyyy-MM-dd HH:mm:ss')}
-            </Trans>
-          </Line>
-        }
-      />
-      <CardActions>
-        <Line expand noMargin justifyContent="flex-end">
-          {!isOld && (
-            <BuildProgressAndActions
-              build={build}
-              game={game}
-              onGameUpdated={onGameUpdated}
-              gameUpdating={gameUpdating}
-              setGameUpdating={setGameUpdating}
-            />
-          )}
-          {isOld && (
-            <EmptyMessage>
-              <Trans>
-                This build is old and the generated games can't be downloaded
-                anymore.
-              </Trans>
-            </EmptyMessage>
+    <ResponsiveWindowMeasurer>
+      {windowWidth => (
+        <I18n>
+          {({ i18n }) => (
+            <>
+              <Card
+                isHighlighted={isOnlineBuild}
+                cardCornerAction={
+                  <ElementWithMenu
+                    element={
+                      <IconButton size="small" disabled={gameUpdating}>
+                        <MoreVert />
+                      </IconButton>
+                    }
+                    buildMenuTemplate={(i18n: I18nType) => [
+                      {
+                        label: i18n._(t`Edit build name`),
+                        click: onEditName,
+                      },
+                      { type: 'separator' },
+                      {
+                        label: i18n._(t`Delete build`),
+                        click: () => onDeleteBuild(i18n),
+                      },
+                    ]}
+                  />
+                }
+                header={
+                  <Line
+                    noMargin
+                    alignItems="start"
+                    justifyContent="space-between"
+                  >
+                    {windowWidth !== 'small' && (
+                      <BuildAndCreatedAt build={build} />
+                    )}
+                    <Column expand noMargin justifyContent="center">
+                      <Line noMargin justifyContent="end">
+                        {isOnlineBuild ? (
+                          <Text size="body2">
+                            <Trans>Current build online</Trans>
+                          </Text>
+                        ) : (
+                          game.acceptsBuildComments &&
+                          build.type === 'web-build' && (
+                            <LineStackLayout alignItems="center" noMargin>
+                              <div
```

Comment:
Any chance we can use UI/Chip? Or is it really different?

Reply:
I think it is totally different, it is for this little green thing over there:
<img width="218" alt="image" src="https://user-images.githubusercontent.com/81410437/176637757-4a9355a8-3e61-4ba0-98a9-19774c27110f.png">